cmd/functional-test/testcases.txt

```diff
@@ -0,0 +1,49 @@
+www.example.com {{binary}} -silent 
+www.example.com {{binary}} -silent -l test-data/request.txt
+www.example.com {{binary}} -silent -request test-data/raw-request.txt
+www.example.com {{binary}} -silent -response-time
+www.example.com {{binary}} -silent -title
+www.example.com {{binary}} -silent -location
+www.example.com {{binary}} -silent -cname
+www.example.com {{binary}} -silent -cdn
+www.example.com {{binary}} -silent -probe
+www.example.com {{binary}} -silent -no-fallback
+www.example.com {{binary}} -silent -match-code 200, -mc 200
```

Comment:
I think we should remove the comma after the status code

Reply:
Done.